### DISCUSSION TITLE
Software Engineer  Intern

### DISCUSSION THREAD

**Discussion by recursivezero** [recursivezero/career#2] [Announcements]
# Job Description: Software Development Intern

**Location:** Remote  
**Hours:** 30 hours per week (Tuesday to Saturday)  
**Daily Meeting:** 8:30 PM status check-in  

---

## About Us

We are a dynamic startup focused on building cutting-edge daily task management tools using the latest technology stacks. Our innovative solutions aim to enhance productivity and streamline workflows. Currently, we have 5 active projects, each leveraging modern tools and frameworks to create impactful user experiences. As we continue to grow, we're looking for talented interns to join our team and contribute to the development of these exciting projects.

---

## Key Responsibilities
- Develop UI components with strong proficiency in HTML, CSS, and JavaScript.
- Work with TypeScript and React (React knowledge is a plus).
- Contribute to AI/ML projects, focusing on speech-to-text and image processing.
- Build and maintain Python-based applications with Streamlit framework experience.
- Collaborate with team members to create clean, concise documentation.
- Actively participate in daily status meetings and contribute as a good team player.

## Requirements
- Strong knowledge of HTML, CSS, and JavaScript.
- Familiarity with TypeScript and React is preferred but not mandatory.
- Basic knowledge of Python, especially in AI/ML, speech-to-text, and image processing techniques.
- Experience with the Streamlit framework is a plus.
- A good understanding of software development fundamentals.
- A strong team player with good documentation skills.
- **Mandatory**: Share your GitHub profile when applying.

## Selection Process
1. Initial task assignment: Complete within 7 days.
2. Evaluation of the task for shortlisting candidates.

## Stipend and Benefits

- Stipend provided, sufficient to cover internet expenses and daily essentials.
- Performance-based increments to recognize and reward your growth.
- Experience certificate provided upon successful completion of the internship.

If you’re excited about this opportunity to work in a fast-paced startup environment then share your GitHub Profile just by saying hi.

>[!NOTE]
> Do not attach cv/resume ; just say hi

**Comment by kavish0302** [comment]
Kavish Gulati Resume. [Resume.pdf](https://github.com/user-attachments/files/17406760/Resume.pdf)

**Comment by boratsbias** [comment]
Anubrat Bora Resume. 
[anubrat-bora-cv.pdf](https://github.com/user-attachments/files/17407382/anubrat-bora-cv.pdf)

**Comment by arthmohan** [comment]
[Arth Resume.pdf](https://github.com/user-attachments/files/17415640/Arth.Resume.pdf)
Arth Mohan Resume

**Comment by Krishika-Garg** [comment]
[Resume_Krishika_Garg.pdf](https://github.com/user-attachments/files/17416731/Resume_Krishika_Garg.pdf)
Krishika Garg Resume

**Comment by Ciphrox** [comment]
Vansh Raj Resume
[Vansh_Cipher_resume.pdf](https://github.com/user-attachments/files/17419423/Vansh_Cipher_resume.pdf)

**Comment by arnavsuman** [comment]
Arnav Suman Resume
[Resume_Arnav_Suman.pdf](https://github.com/user-attachments/files/17421414/Resume_Arnav_Suman.pdf)

**Comment by Zem-0** [comment]
Parinith
[Parinith Resume.pdf](https://github.com/user-attachments/files/17421546/Parinith.Resume.pdf)

**Comment by AdityaSingh-7** [comment]
Aditya Kumar Singh
resume- 

[Aditya_Singh_resume.pdf](https://github.com/user-attachments/files/17428831/Aditya_Singh_resume.pdf)
